tracer/test/snapshots/NetActivitySdkTests.verified.txt

```diff
@@ -0,0 +1,456 @@
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_9,
+    Name: Samples.NetActivitySdk.internal,
+    Resource: UnsetStatusSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_8,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_10,
+    Name: Samples.NetActivitySdk.internal,
+    Resource: ParentSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_9,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_11,
+    Name: opentelemetry.internal,
+    Resource: ChildSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_10,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: ,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_12,
+    Name: Samples.NetActivitySdk.internal,
+    Resource: W3CParentSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_10,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_13,
+    Name: opentelemetry.internal,
+    Resource: W3CChildSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_12,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: ,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_14,
+    Name: Samples.NetActivitySdk.internal,
+    Resource: IAmMiscSpan,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_12,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_3,
+      span.kind: internal
+    }
+  },
+  {
+    TraceId: Id_6,
+    SpanId: Id_15,
+    Name: opentelemetry.internal,
+    Resource: MiscSpan2,
+    Service: Samples.NetActivitySdk,
+    Type: custom,
+    ParentId: Id_14,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: ,
```

Comment:
I can't remember if this was a bug we fixed, but with this is incorrectly missing, or we shouldn't add the property I think 🤔

Reply:
I thought I fixed this in my PR 🤔 Maybe the snapshot isn't up-to-date?

Reply:
So I think I see what is happening here

Basically, since this was just a `new Activity` that I called `.Start` on the `Activity.Source.Name` doesn't ever get set it seems to only be set when created through the `ActivitySource.StartActivity`. 

I'll check to see if the `Source.Name` has a value and if it doesn't I won't include `otel.library.name` - I think that is reasonable